Automattic/newspack-plugin
# Tier support for Donations wizard

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds support for tiers to the donations wizard:

![localhost_wp-admin_admin php_page=newspack-donations-wizard (1)](https://user-images.githubusercontent.com/7317227/61741155-1c42cf80-ad45-11e9-946f-79f37efb7781.png)
![localhost_wp-admin_admin php_page=newspack-donations-wizard (2)](https://user-images.githubusercontent.com/7317227/61741157-1e0c9300-ad45-11e9-9321-c7e4cd5f3bd4.png)

The donation block should be able to get all the info it needs through the `/newspack/v1/wizard/newspack-donations-wizard/donation` endpoint, including whether tiers are used and the tier settings.

### How to test the changes in this Pull Request:

1. Go to Donations wizard. Test that the tier-based and non-tier-based donation settings save/load well. You won't be able to see the low/high tiers on the product frontend, but the middle tier should be the  suggested price.

### Other information:
